.github/workflows/submitBeta.yml

```diff
@@ -0,0 +1,127 @@
+name: Submit extension to browser store
+on:
+  workflow_dispatch:
+    inputs:
+      release:
+        description: Release Number
+        default: "2.0.0"
+        required: true
+      version:
+        description: Beta version
+        default: "0"
+        required: true
+jobs:
+  bump-version:
+    name: Bump Package Version and Submit Extension
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          ref: release/${{ github.event.inputs.release }}
+      - name: Update manifest.json name
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: name
+          value: Freighter BETA
+      - name: Update manifest.json version_name
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: version_name
+          value: ${{ github.event.inputs.release }}-beta.${{ github.event.inputs.release }}
+      - name: Update manifest.json version
+        uses: jossef/action-set-json-field@6e6d7e639f24b3955ef682815317b5613ac6ca12 #v1
+        with:
+          file: ./extension/public/static/manifest.json
+          field: version
+          value: ${{ github.event.inputs.release }}
+      - name: Build extension
```

Comment:
❓ I noticed the `submitProduction.yml` doesn't have this `Build extension for Firefox` command, but submitBeta does, even though looks like both are submitting to firefox, is that on purpose?